WBooks/New Group/Library/LibraryViewModel.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  LibraryViewModel.swift
+//  WBooks
+//
+//  Created by Fermin Robilotte on 30/04/2019.
+//  Copyright © 2019 Wolox. All rights reserved.
+//
+
+import Foundation
+
+class LibraryViewModel {
+    private let _repository: BookRepository
+    var arrayBooks: [BookViewModel] = []
+    
+    init (repository: BookRepository) {
+        _repository = repository
+    }
+    
+    func getBooks(reloadTable: @escaping () -> Void) {
+        _repository.fetchBooks(onSuccess: { (books) in
```

Comment:
dont forget to add `[unowned self] books in` at the start of the closure to avoid retaining self